clang/test/CodeCompletion/ordinary-name-cxx11.cpp

```diff
@@ -57,7 +58,6 @@ void foo() {
   // CHECK-CC1: COMPLETION: TYPEDEF : TYPEDEF
   // CHECK-CC1-NEXT: COMPLETION: Pattern : typedef <#type#> <#name#>;
   // CHECK-CC1-NEXT: COMPLETION: Pattern : [#std::type_info#]typeid(<#expression-or-type#>)
-  // CHECK-CC1-NEXT: COMPLETION: Pattern : typename <#name#>
```

Comment:
this was failin before I added my changes. somehow the initial Coroutine PR broke it. see:
https://github.com/FlyingsMarmot/llvm-project/actions/runs/11867203529/job/33074965044#step:7:5562